src/datapump/jobs/geotrellis.py

```diff
@@ -821,8 +821,7 @@ def _run_job_flow(self, name, instances, steps, applications, configurations):
                 {
                     "Name": "Install GDAL",
                     "ScriptBootstrapAction": {
-                        "Path": f"s3://{GLOBALS.s3_bucket_pipeline}/geotrellis/bootstrap/gdal.sh",
-                        "Args": ["3.1.2"],
+                        "Path": f"s3://{GLOBALS.s3_bucket_pipeline}/geotrellis/bootstrap/gdal-3.8.3.sh"
```

Comment:
small nitpick, but you could reduce the lines of code by just having a block before this like:

```
bootstrap_path = f"s3://{GLOBALS.s3_bucket_pipeline}/geotrellis/bootstrap/gdal-3.8.3.sh"
if self.geotrellis_version < "2.4.1":
    f"s3://{GLOBALS.s3_bucket_pipeline}/geotrellis/bootstrap/gdal.sh"
```

And then this line would just be changed to:
```
"Path": bootstrap_path
```